lib/Search/ElasticSearch/ElasticSearchEngine.php

```diff
@@ -135,8 +160,10 @@ private function createSearchParams($query)
                 ],
             ],
         ];
+    }
 
-        return $params;
+    private function endsWith($haystack, $needle) {
```

Comment:
@Dillon-Brown @mattlorimer this method was added but does not seem to be used.

Maybe it was supposed to be used in the following?

```php
        if ((!substr_compare($searchStr, $wildcardBe, -strlen($wildcardBe))) === 0) {
            $searchStr .= $wildcardBe;
        }
```

Reply:
Ahh yeah, this can be removed. I was looking to see if we had a similar method in the utils or something and must have copied this in. We only need to do this once so we don't really need a dedicated method here. Pushed!

Reply:
Thanks!